### DISCUSSION TITLE
Changing the Width/Style of the Dialog

### DISCUSSION THREAD

**Discussion by codename000** [MahApps/MahApps.Metro#4185] [Q&A]
I would like to change the look of the dialog so it looks more like a dialog box rather than the background of it spanning across the screen.
![image](https://user-images.githubusercontent.com/10926938/133958005-db56cbba-3b11-4468-b569-d0712f732d57.png)

I've managed to override the style so it works on ShowMessageAsync (see above), but other methods like ShowLoginAsync or ShowModalDialogExternal remain unchanged somehow.
![image](https://user-images.githubusercontent.com/10926938/133958040-10a3e72c-de2c-40d8-a842-6017eaebe117.png)

Do i have to create a custom content template for each types of the dialog? Can anyone shed some light on this please? Thanks in advance.

**Comment by punker76** [comment]
@codename000 In the current source there was a change to allow easily extent the dialog styles. If this doesn't help then you can try an alternative to the MahApps dialogs with the [SimpleChildWindow](https://github.com/punker76/MahApps.Metro.SimpleChildWindow)
